src/main/java/lotto/util/Parser.java

```diff
@@ -0,0 +1,35 @@
+package lotto.util;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static lotto.constant.ErrorMessage.*;
+
+public class Parser {
+    public static int stringToInt(String string) {
+        try {
+            return Integer.parseInt(string);
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException(INVALID_NUMBER_FORMAT_ERROR.getErrorMessage());
+        }
+    }
+
+    public static List<Integer> stringToInt(List<String> parsedWinningNumber) {
+        try {
+            return parsedWinningNumber.stream()
+                    .map(Integer::parseInt)
+                    .toList();
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException(INVALID_NUMBER_FORMAT_ERROR.getErrorMessage());
+        }
+    }
+
+    public static List<String> parseWinningNumber(String inputWinningNumber) {
+        List<String> parsedWinningNumber = new ArrayList<>();
+        Arrays.stream(inputWinningNumber.split(","))
+                .map(String::trim)
+                .forEach(parsedWinningNumber::add);
+        return parsedWinningNumber;
+    }
```

Comment:
```suggestion
    public static List<String> parseWinningNumber(String inputWinningNumber) {
        return Arrays.stream(inputWinningNumber.split(","))
                .map(String::trim)
                .toList();
    }
```
이렇게 바꿔주시면 스트림을 조금 더 효율적으로 사용할 수 있을 것 같습니다! 추가로 `toList()` 덕분에 불변성 보장도 가능합니다.